src/site/layouts/health_care_region_page.drupal.liquid

```diff
@@ -138,73 +140,115 @@ Example data:
-      <h2>Manage your health online</h2>
-      <div class="usa-grid usa-grid-full">
-        <div class="usa-width-one-half">
-          Refill and track your prescriptions
-        </div>
-        <div class="usa-width-one-half">
-          Get you VA medical records (Blue Button)
-        </div>
-      </div>
-      <div class="usa-grid usa-grid-full">
-        <div class="usa-width-one-half">
-          Send a secure message to your health care team
-        </div>
-        <div class="usa-width-one-half">
-          View your lab and test results
+        <div class="usa-font-lead">
+          <p>{{ fieldIntroText }}</p>
         </div>
-      </div>
-      <div class="usa-grid usa-grid-full">
-        <div class="usa-width-one-half">
-          Schedule and view your appointments
-        </div>
-        <div class="usa-width-one-half">
-          Order hearing aid batteries and prosthetic socks
+
+        <section>
+          <h2>Main Locations</h2>
+          <div data-widget-type="basic-facility-locations-list" data-facilities="{{ mainFacilities.entities | widgetFacilitiesList | escape }}" data-path="{{ entityUrl.path }}"></div>
+          <a class="usa-button usa-button-secondary vads-facility-hub-button" href="{{ entityUrl.path }}/locations">
+            See all locations <i class="fa fa-chevron-right">&nbsp;</i>
+          </a>
+        </section>
+
+        <!-- "Manage your health online" section -->
+        <section>
+          <h3>Manage your health online</h3>
+          <div class="usa-grid usa-grid-full">
+            <div class="usa-width-one-half vads-facility-hub-cta">
+              <a href="https://www.va.gov/health-care/refill-track-prescriptions/">
+                <i class=" fa fa-prescription-bottle vads-facility-hub-cta-circle">&nbsp;</i>
+                <span class="vads-facility-hub-cta-label">Refill and track your prescriptions <i class="fa fa-chevron-right vads-facility-hub-cta-arrow">&nbsp;</i></span>
+              </a>
+            </div>
+            <div class="usa-width-one-half vads-facility-hub-cta">
+            <a href="https://www.va.gov/health-care/get-medical-records/">
+              <i class="fa fa-file-medical vads-facility-hub-cta-circle">&nbsp;</i>
+              <span class="vads-facility-hub-cta-label">Download your VA medical records (Blue Button) <i class="fa fa-chevron-right vads-facility-hub-cta-arrow">&nbsp;</i></span>
+            </a>
+            </div>
+          </div>
+          <div class="usa-grid usa-grid-full">
+            <div class="usa-width-one-half vads-facility-hub-cta">
+              <a href="https://www.va.gov/health-care/secure-messaging/">
+                <i class="fa fa-comments vads-facility-hub-cta-circle">&nbsp;</i>
+                <span class="vads-facility-hub-cta-label">Send a secure message to your health care team <i class="fa fa-chevron-right vads-facility-hub-cta-arrow">&nbsp;</i></span>
+              </a>
+            </div>
```

Comment:
I think we can omit `https://www.va.gov` and just use the leading `/` to create links to internal pages. This way we preserve the environment's subdomain.